.github/workflows/build-test-push.yml

```diff
@@ -0,0 +1,76 @@
+# This workflow is based on
+# https://docs.github.com/en/actions/publishing-packages/publishing-docker-images.
+
+name: Build, test, and push image
+
+on:
+  pull_request:
+  push:
+  workflow_dispatch:
+
+env:
+  REGISTRY: ghcr.io
```

Comment:
Now that we change to using ghcr,io instead of nordix we should modify the README as well, https://github.com/eiffel-community/eiffel-goer/blob/main/README.md?plain=1#L20

Also, can we remove https://github.com/eiffel-community/eiffel-goer/blob/6233cbba455bf4167394a4c096ec0ed1912d8903/Makefile#L61-L68?

Reply:
Good catch, totally forgot about the readme. I'm somewhat inclined to keep the `docker` makefile rule (after changing how the image is tagged) for improved quality of life when testing locally built containers.

Reply:
Yes, suggesting removing the docker make target was a lil too aggressive.

Reply:
Done.